addon/defaults.js

```diff
@@ -0,0 +1 @@
+export const DEFAULT_NETWORK_CODE = '6483581'
```

Comment:
Can you double check that you're getting the expected value for `DEFAULT_NETWORK_CODE` with this syntax? This is a named export, i.e. not using the `default` keyword, which I would expect to be imported like this:
```js
import { DEFAULT_NETWORK_CODE } from 'nypr-ads/defaults';
```

Reply:
Ah, good catch.